openmc/cmfd.py

```diff
@@ -7,9 +7,53 @@
 
 
 class CMFDMesh(object):
+    """A structured Cartesian mesh used for coarse mesh finite difference
```

Comment:
I would suggest capitalizing "Coarse Mesh Finite Difference (CMFD)" here just to make it painfully obvious where acronym in the class name comes from.